pkg/util/cluster/aad.go

```diff
@@ -6,57 +6,41 @@ package cluster
 import (
 	"context"
 	"fmt"
-	"net/http"
 	"time"
 
-	azgraphrbac "github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
-	"github.com/Azure/go-autorest/autorest"
-	"github.com/Azure/go-autorest/autorest/date"
+	msgraph_apps "github.com/microsoftgraph/msgraph-sdk-go/applications"
+	msgraph_models "github.com/microsoftgraph/msgraph-sdk-go/models"
+	msgraph_errors "github.com/microsoftgraph/msgraph-sdk-go/models/odataerrors"
 	"k8s.io/apimachinery/pkg/util/wait"
-
-	"github.com/Azure/ARO-RP/pkg/util/uuid"
 )
 
-func (c *Cluster) getServicePrincipal(ctx context.Context, appID string) (string, error) {
```

Comment:
Interesting that we don't need this anymore, nice!

Reply:
It's basically the same as `GetServicePrincipalIdByAppID`.